.i18nrc.json

```diff
@@ -25,6 +26,8 @@
     "src/ui/ui_render/bootstrap/app_bootstrap.js",
     "src/ui/ui_render/ui_render_mixin.js",
     "x-pack/plugins/monitoring/public/components/cluster/overview/alerts_panel.js",
-    "x-pack/plugins/monitoring/public/directives/alerts/index.js"
+    "x-pack/plugins/monitoring/public/directives/alerts/index.js",
+    "x-pack/plugins/infra/public/utils/loading_state/loading_result.ts",
```

Comment:
These files were excluded because `i18n` validation tool can not parse them due to an [issue](https://github.com/babel/babel/issues/7742) in babel parser which is used by the tool.